tools/cldr-code/src/main/java/org/unicode/cldr/util/personname/PersonNameFormatter.java

```diff
@@ -1297,6 +1309,31 @@ public PersonNameFormatter(CLDRFile cldrFile) {
         this.fallbackFormatter = new FallbackFormatter(new ULocale(cldrFile.getLocaleID()), initialPattern, initialSequencePattern);
     }
 
+    /**
+     * Utility for constructing data from path and value.
+     */
+    public static Pair<ParameterMatcher, NamePattern> fromPathValue(XPathParts parts, String value) {
+
+        //ldml/personNames/personName[@length="long"][@usage="sorting"]/namePattern[alt="2"]
```

Comment:
Since @usage="sorting" is not longer valid, might want to change this comment to use e.g. @usage="referring"

Reply:
done

Reply:
Actually the comment still has @usage="sorting" but we can fix in a future PR